homeassistant/components/anthemav/media_player.py

```diff
@@ -44,39 +46,41 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
     }
 )
 
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up our socket to the AVR."""
+    _LOGGER.warning(
+        "AnthemAV configuration is deprecated and has been automatically imported. Please remove the integration from your configuration file"
+    )
+    await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_IMPORT},
+        data=config,
+    )
 
-    host = config[CONF_HOST]
-    port = config[CONF_PORT]
-    name = config.get(CONF_NAME)
-    device = None
 
-    _LOGGER.info("Provisioning Anthem AVR device at %s:%d", host, port)
+async def async_setup_entry(hass, config_entry, async_add_devices):
```

Comment:
```suggestion
async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry, async_add_devices: AddEntitiesCallback) -> None:
```
Seems as typing is missing throughout